### PR TITLE
Feature: new entry point for events, commands and async queries handler.

### LINKED ISSUE #100: New event subscriptions entry point
When I need an event subscriber as an entry point I need to create a generic entry point and modified it adding reactive commons and creating the handlers and listeners.  The plugin should do it for me!

### PR DESCRIPTION
**Pull request by juancgalvis**
## Description
Implements new Entry Point `ASYNCEVENTHANDLER` for events, commands and async queries using Reactive Commons abstraction. Resolves #100

Driven Adapter `ASYNCEVENTBUS` updated to be able to send commands and async queries.

## Category
- [X] Feature
- [ ] Fix

## Checklist
- [X] The pull request is complete according to the [guide of contributing](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing)
- [X] Automated tests are written
- [X] The documentation is up-to-date
- [X] the version of the build.gradle, readme.md and gradle.properties was increased
- [X] The pull request has a descriptive title that describes what has changed, and provides enough context for the changelog

### CODE REVIEW

File: src/main/resources/driven-adapter/async-event-bus/reactive-direct-async-gateway.java.mustache [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import org.reactivecommons.async.impl.config.annotations.EnableDirectAsyncGateway;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+{{#lombok}}
+import java.util.logging.Level;
+{{/lombok}}
+
+{{#lombok}}
+@Log
```
Comment (santitigaga) [inline src/main/resources/driven-adapter/async-event-bus/reactive-direct-async-gateway.java.mustache:19]: maybe if you change the order yo need only one lombok validation

File: src/main/resources/driven-adapter/async-event-bus/reactive-events-gateway.java.mustache [hunk 2]
```diff
@@ -1,45 +1,42 @@
...
+{{/lombok}}
 
 import static reactor.core.publisher.Mono.from;
 
-@Component
-@EnableDomainEventBus
 {{#lombok}}
 @Log
 @RequiredArgsConstructor
 {{/lombok}}
```
Comment (santitigaga) [inline src/main/resources/driven-adapter/async-event-bus/reactive-events-gateway.java.mustache:23]: maybe if you change the order yo need only one lombok validation

File: src/main/resources/entry-point/async-event-handler/commands-handler.java.mustache [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package {{package}}.events.handlers;
+
+{{#lombok}}
```
Comment (santitigaga) [inline src/main/resources/entry-point/async-event-handler/commands-handler.java.mustache:3]: maybe if you change the order yo need only one lombok validation

File: src/main/resources/entry-point/async-event-handler/events-handler.java.mustache [hunk 4]
```diff
@@ -0,0 +1,28 @@
+package {{package}}.events.handlers;
+
+{{#lombok}}
```
Comment (santitigaga) [inline src/main/resources/entry-point/async-event-handler/events-handler.java.mustache:3]: maybe if you change the order yo need only one lombok validation

File: src/main/resources/entry-point/async-event-handler/queries-handler.java.mustache [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package {{package}}.events.handlers;
+
+{{#lombok}}
```
Comment (santitigaga) [inline src/main/resources/entry-point/async-event-handler/queries-handler.java.mustache:3]: maybe if you change the order yo need only one lombok validation

**Review by santitigaga — APPROVED** [review]

File: src/main/resources/driven-adapter/async-event-bus/reactive-events-gateway.java.mustache (hunk 2 above)
Comment (juancgalvis) [reply src/main/resources/driven-adapter/async-event-bus/reactive-events-gateway.java.mustache:23]: could be, but the imports are sorted and arranged by package

**Pull request merged**